thoth/storages/graph/postgres.py

```diff
@@ -3652,7 +3652,7 @@ def get_python_package_version_metadata(
         if result is None:
             raise NotFoundError(f"No record found for {package_name!r}, {package_version!r}, {index_url!r}")
 
-        return result.to_dict()
+        return dict(zip([col.name for col in selected_columns], result))
```

Comment:
You can remove `with_entities` part and call `to_dict()` on the retrieved model. No need to have `selected_columns`.

Reply:
Corrected, thanks :)